bindings/c/ParameterFramework.cpp

```diff
@@ -148,9 +148,7 @@ PfwHandler *pfwCreate()
 
 void pfwDestroy(PfwHandler *handle)
 {
-    if (handle->pfw != NULL) {
-        delete handle->pfw;
-    }
+    delete handle->pfw;
```

Comment:
if handle--> is expected non null at Destroy, all tests on others methods against this may be dropped too